### ISSUE TITLE
Client: Draft conversion guides

### ISSUE THREAD

**Issue by TiE23** [TiE23/metric-teacher#216]
This site is about teaching people how to convert from one system of units to another. When I was using Duolingo I grew annoyed over time that the mobile app didn't contain any guidance for the more nuanced language rules surrounding the language I was studying. So while the repetitious nature of the learning tool was great for the start I ran out of steam as confusing rules around verb conjugation and numerous particles were left unexplained and it was expected of me to recognize very sparsely displayed patterns in my lessons.

So, I want to avoid that issue by providing a few tools:

* A summary of the Metric and Imperial systems. Their history, how they're constructed, how they're related, and the strengths and weaknesses in each one.
* A dedicated page with prose explanations for each unit with the following details:
  * The story behind each one.
  * How they related to units in their system (if any).
* A chart of conversion values.
  * This would be accessible in Challenge mode as a pop-up modal. It would contain a table of conversion values and any other very important details that a student would want to reference quickly in the middle of a challenge.

For this ticket decide on what each section is and then draft up the written details. Create a new ticket to actually provide the resources on the website. (This might be one or two tickets as there are really two parts: a library part of the site and a conversion/reference modal pop-up part for Challenge mode).

**Comment by TiE23** [comment]
Big loving shoutout to Daryl for finding the source of the React-Router's API page scrolling magic:
https://github.com/ReactTraining/react-router/blob/master/website/modules/components/ScrollToDoc.js

> Yo! Seeking advice for the first time since leaving like 9 months ago. I cannot Google my way into understanding how to do the same kind of scrolling that the API docs for React-Router does.
> 
> How a link from: https://reacttraining.com/react-router/web/api/Redirect
> Easily scrolls to: https://reacttraining.com/react-router/web/api/Redirect/to-string
> 
> It's slick and clean, it's all in the route, and I want it for my own site. I don't wanna use hash links if I can avoid it. And since it's the API doc for react-router I just assume that it's done with vanilla react-router (I mean, that's only what they'd, right?).
> 
> If you just know the answer or can give me the specific name for this thing to Google or can link me to a guide on how to do this I'd really appreciate it.
> 
> If not, no biggie, it's not a high priority for me (it's a nice-to-have at the moment). But I am willing to bet it's a pretty simple thing to pull off (the number of times in the last six months where something complicated ended up being simple as hell gives me hope).

**Comment by TiE23** [comment]
Oh, whoops. Should've closed this yesterday before starting #223
